website/docs/language/upgrade-guides/index.mdx

```diff
@@ -33,7 +33,7 @@ Executing `terraform init -reconfigure` is required after updating to Terraform
 
 ### S3 Native State Locking
 
-The S3 backend now supports S3 native state locking as an opt-in, experimental feature.
+The S3 backend supports S3 native state locking as an opt-in feature.
```

Comment:
This page typically gets wiped and re-written by the core team before the formal minor release date. Perhaps we omit this for now and start a new entry once the other bits from core are added?